spec/php-spec.coffee

```diff
@@ -264,6 +264,27 @@ describe 'PHP grammar', ->
         expect(tokens[1][15]).toEqual value: '{', scopes: ['text.html.php', 'meta.embedded.block.php', 'source.php', 'punctuation.section.scope.begin.php']
         expect(tokens[1][16]).toEqual value: '}', scopes: ['text.html.php', 'meta.embedded.block.php', 'source.php', 'punctuation.section.scope.end.php']
 
+      it 'should tokenize a non-empty array correctly', ->
+        tokens = grammar.tokenizeLines "<?php\nfunction not_empty_array_test(array $value = [1,2,'3']) {}"
+
+        expect(tokens[1][11]).toEqual value: '[', scopes: ['text.html.php', 'meta.embedded.block.php', 'source.php', 'meta.function.php', 'meta.function.arguments.php', 'meta.function.argument.array.php', 'punctuation.section.array.begin.php']
+
```

Comment:
Last thing to do: these newlines aren't needed.  Take a look at the other tests for an example.

eg
```
expect...
expect...
```
instead of
```
expect...

expect...
```